target/linux/rockchip/armv8/config-6.12

```diff
@@ -424,13 +423,15 @@ CONFIG_MTD_CMDLINE_PARTS=y
 CONFIG_MTD_SPI_NOR=y
 CONFIG_MTD_SPI_NOR_USE_4K_SECTORS=y
 CONFIG_MTD_SPLIT_FIRMWARE=y
+CONFIG_MTK_NET_PHYLIB=y
 CONFIG_MUTEX_SPIN_ON_OWNER=y
 CONFIG_NEED_DMA_MAP_STATE=y
 CONFIG_NEED_SG_DMA_FLAGS=y
 CONFIG_NEED_SG_DMA_LENGTH=y
 CONFIG_NET_DEVLINK=y
 CONFIG_NET_DEVMEM=y
 CONFIG_NET_DSA=y
+# CONFIG_NET_DSA_KS8995 is not set
```

Comment:
Please move this to `target/linux/generic/config-6.12`. It was added in 73504d0b27847e70f0cdb2acddd0599c7c9c6cc2

Reply:
fixed